gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenCompoundsMapper.java

```diff
@@ -230,19 +230,6 @@ private void handleBatteryCompoundStatement(RCMRMT030101UKCompoundStatement spec
             );
         }
 
-        batteryCompoundStatement.getComponent().stream()
```

Comment:
So on second look, line 224 has a filter "CLUSTER_CLASSCODE". So the duplication _only_ occurrs when the battery has clusters inside it.